### DISCUSSION TITLE
SQL Server resiliency: Any plans to move to entity framework core or use System.Data.SQLClient.SQLConnection?

### DISCUSSION THREAD

**Discussion by mkredpoint** [quartznet/quartznet#535] [General] [answered]
We're looking to start a Quartz.net project using a SQL Azure DB as the persistence store. Azure SQL DB is notorious for occasional flickers where the serviced goes down as a patch or some maintenance goes on in Azure. The MS teams have been mitigating this in newer versions of Entity Framework and .NET core:
https://msdn.microsoft.com/en-us/library/dn456835(v=vs.113).aspx
https://docs.microsoft.com/en-us/azure/sql-database/sql-database-connectivity-issues

Is providing better resiliency to cloud db providers anywhere on the radar for this project?

Thanks

**Comment by lahma** [comment, marked as answer]
Quartz.NET should run quite fine on Azure. There are both retries and transient fault detections in place for problematic cases. It's more like retry on all errors kind of retry, but should cover the Azure case too.

SqlConnection is used for SQL Server access, so if you are running on .NET 4.6.1 you can configure the parameters to your connection string the way you like.

Please let us know if you have any trouble running Quartz.NET against Azure DB, many are already doing that.
